README.md

```diff
@@ -79,27 +86,6 @@ Insert a new value. Will merge any previous values seen for this key.
 `callback`'s parameters are `err, node`, where `node` is the newly inserted
```

Comment:
FYI this new code no longer includes `node` in its callback, which is a semver break as-is.